quickstart_envs.sh

```diff
@@ -3,7 +3,7 @@
 # example minimal set of environment variables to get started - see readme for additional envs you may wish to set
 
 # embedded tomcat LabKey .jar version to build container with
-export LABKEY_VERSION="24.11"
+export LABKEY_VERSION="25.3-SNAPSHOT"
```

Comment:
outside users don't have access to the snapshots.

```suggestion
export LABKEY_VERSION="25.3"
```

Reply:
updated